src/main/java/com/waratek/spiracle/sql/util/SelectUtil.java

```diff
@@ -128,7 +128,6 @@ public static void executeQuery(String sql, ServletContext application, HttpServ
 					logger.error(conCloseException);
 				}
 			}
-			TagUtil.printPageFooter(out);
```

Comment:
Didn't notice this was duplicated. Removal is on the wrong place though. if writeToResponse throws an exception the footer will not be printed since the finally block is the the only part that is guaranteed to be executed.